### PR TITLE
extracted 'common.adrress' to new 'address' package

### PR DESCRIPTION
**Pull request by alonp99**
https://github.com/spacemeshos/go-spacemesh/issues/410

### CODE REVIEW

**Comment by alonp99** [comment]
@antonlerner :)

File: .gitignore [hunk 1]
```diff
@@ -26,4 +26,5 @@
 **/pb/*.pb.go
 **/pb/*.pb.gw.go
 **/pb/*.swagger.json
-database/data
\ No newline at end of file
+database/data
+app/state
```
Comment (antonlerner) [inline .gitignore:30]: remove this one but don't commit any blobs from here

**Review by antonlerner — APPROVED** [review]

**Comment by antonlerner** [comment]
Please update this branch and I'll merge it

**Comment by antonlerner** [comment]
Please run tests... as you can see they get stuck, probably on sync tests - maybe theres a loop somewhere

File: .gitignore (hunk 1 above)
Comment (alonp99) [reply .gitignore:30]: I see it was already merged, did you make the change?

**Pull request merged**